packages/async-flow/src/async-flow.js

```diff
@@ -452,6 +453,7 @@ export const prepareAsyncFlowTools = (outerZone, outerOptions = {}) => {
     defineProperties(wrapperFunc, {
       length: { value: guestFunc.length },
     });
+    // @ts-expect-error cast
```

Comment:
curious, why not cast instead? actually couldn't we set the type at the definition above?
```js
    const wrapperFunc = {
      /** @type {HostOf<F>} */
      [hostFuncName](...args) {

```

Reply:
better!